cfunits/units.py

```diff
@@ -344,6 +345,46 @@ def add_unit_alias(definition, symbol, singular, plural):
 #      [-99, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]])
 
 
+# --------------------------------------------------------------------
+# Function to decode Udunits status codes to text
+# --------------------------------------------------------------------
+def decode_status(status):
```

Comment:
```suggestion
def _decode_status(status):
```
... and doc string, please!

Reply:
Can you provide an example of when this would ever get called? I see that we only ever call `_ut_format` if there already an integer `self._ut_unit`, so I'm wondering how the former could go wrong?